src/main.rs

```diff
@@ -82,7 +82,8 @@ fn run(cli: Cli) -> Result<(), Error> {
     }
 
     // We're going to need a source of randomness.
-    let mut rng = rand::thread_rng();
+    let mut rng =
+        cli.seed.map(rand::rngs::StdRng::seed_from_u64).unwrap_or_else(|| rand::rngs::StdRng::from_entropy());
```

Comment:
This can be simplified:

```suggestion
        cli.seed.map(rand::rngs::StdRng::seed_from_u64).unwrap_or_else(rand::rngs::StdRng::from_entropy);
```